Formula/vim@7.4.rb

```diff
@@ -77,9 +77,18 @@ def install
       opts << "--without-x"
     end
 
-    if build.with? "luajit"
```

Comment:
Because the idea of versioned formulae is not to give the same level of support as the main formulae, and major rewrites like this violate that.